Src/Notion.Client/Models/Database/Properties/RollupProperty.cs

```diff
@@ -70,6 +70,12 @@ public enum Function
         Max,
 
         [EnumMember(Value = "range")]
-        Range
+        Range,
+
+        [EnumMember(Value = "show_original")]
+        ShowOriginal,
+
+        [EnumMember(Value = "show_unique")]
```

Comment:
@AnkurSheel Thank you for raising the PR. One thing though I checked the API document but I did not see the `show_unique` enum. Rest of the changes LGTM.

https://developers.notion.com/reference/database#rollup-configuration.

Reply:
It is one of the options when configuring rollups
![image](https://user-images.githubusercontent.com/3807463/127642558-c8bc7c4f-1eac-4ef0-9cc9-50b9f7d97403.png)